docs/source/faq.rst

```diff
@@ -332,3 +332,37 @@ Note that the above examples are similar to running the garbage collector inside
 
     :class:`~optuna.integration.ChainerMNStudy` does currently not provide ``gc_after_trial`` nor callbacks for :func:`~optuna.integration.ChainerMNStudy.optimize`.
     When using this class, you will have to call the garbage collector inside the objective function.
+
+How do I suggest variables which represent the proportion, that is, are in accordance with Dirichlet distribution?
+------------------------------------------------------------------------------------------------------------------
+
+When you want to suggest `n` variables which represent the proportion, that is, `p[0], p[1], ..., p[n-1]` which satisfy `0 <= p[k] <= 1` for any `k` and `p[0] + p[1] + ... + p[n-1] = 1`, try the following.
+These variables are in accordance with the Dirichlet distribution.
+
+.. code-block:: python
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import optuna
+    ​
+
+    def objective(trial):
+        n = 5
+        x = []
+        for i in range(n - 1):
```

Comment:
? 
```suggestion
        for i in range(n:
```

Since https://github.com/optuna/optuna/pull/2718/files#diff-399cec32862278b1cf76d3019a66416c9c5c00e55a067554a69ebebba7684ef3R357 cannot access the last `i`.